src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -372,7 +372,9 @@ export class FindWidget extends Widget implements IOverlayWidget, IHorizontalSas
 		} else {
 			label = NLS_NO_RESULTS;
 		}
-		this._matchesCount.appendChild(document.createTextNode(label));
+		let myAlert = document.createElement('div');
```

Comment:
Why do we need to introduce a `div` element? Can you please explain that.

Reply:
Also if we decide this this is need `myAlert` should be a `const` not a `let`

Reply:
> Why do we need to introduce a `div` element? Can you please explain that.

1. aria-live - seems to need a change, beyond just a text update. Without the DIV, the reading of aria-live is not consistent
2. I investigated to review the accessibility tree - for clues as to why, but it does not seem that there is anyway to access the 
*. chrome://accessibility/
**. accessibility tab - in the Element inspector is not available

I will try to install this tool - to review it for any clues 
https://electronjs.org/docs/tutorial/accessibility#devtron

Reply:
the extra div element - is needed, since normal text changes to the DOM are not enough 

https://github.com/microsoft/vscode/blob/master/src/vs/base/browser/ui/aria/aria.ts#L58